src/providers/wms/qgswmsdataitems.cpp

```diff
@@ -488,6 +496,38 @@ void QgsXyzTileRootItem::newConnection()
   QgsXyzConnectionUtils::addConnection( dlg.connection() );
   refreshConnections();
 }
+
+void QgsXyzTileRootItem::addDefaultXYZServers()
+{
+  QMap<QString, QString> defaultXYZServers;
+  defaultXYZServers[QStringLiteral( "OpenStreetMap Mapnick" )] = QStringLiteral( "http://tile.openstreetmap.org/{z}/{x}/{y}.png" );
```

Comment:
what about to have all these defaults get from the global settings of QgsSettings? in this way QGIS package can be customised basing on what is the target.